README.md

```diff
@@ -15,7 +15,8 @@ It is not a requirement for using JSON context logging.
 
  Laravel             | Package
 :--------------------|:--------
- 6.0 and up          | 1.0
+ 6.0 - 8.0           | 1.0
+ 9.0 and up          | 1.1
```

Comment:
I spent some time investigating the issue here, it's not a Laravel 9 or a PHP 8.1 issue.

This PR makes the package compatible with https://packagist.org/packages/psr/log versions 2 and 3, right now  it's only compatible with version 1 of the psr.

It is indeed a breaking change. 

I have updated the read me for the version number (not sure if the wording is fine).

I have also updated composer.json to hard require version 2 or 3.

I think this change is needed because there are other packages in the wild like (bugsnag-psr-logger) that require psr log 2.0 and up.

Thanks

Reply:
Right, that makes way more sense. 👍